src/components/volume/volume.js

```diff
@@ -389,6 +389,7 @@ class Volume extends Component {
    * @memberof Volume
    */
   render(): React$Element<any> {
+    if (this.props.player.isImg()) return undefined;
```

Comment:
what will happen for switch media? will you be able to restore the volume control?

Reply:
yes i tested it